### ISSUE TITLE
Issue with css keyframes with percentage selectors

### ISSUE THREAD

**Issue by bgriffith** [tonyganch/gonzales-pe#113]
Currently there is a parse error when you use a percentage selector within a keyframe animation.

This is with 3.0.3.

``` css
@keyframes anim {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}
```

While the following is ok

``` css
@keyframes anim {
  from {
    opacity: 0;
  }
  to {
    opacity: 1;
  }
}
```

**Comment by jwilsson** [comment]
I'm seeing the same error for Less.

**Comment by tonyganch** [comment]
I'm working at this issue right now and I have a question for you.
Here is a simplified parse tree for the example above:

```
 -> stylesheet
 | -> atrule
 | | -> atkeyword
 | | | -> ident
 | | |    "keyframes"
 | | -> space
 | |    " "
 | | -> ident
 | |    "anim"
 | | -> space
 | |    " "
 | | -> block
 | | | -> space
 | | |    "\n  "
 | | | -> ruleset
 | | | | -> keyframesSelector
 | | | | | -> percentage
 | | | | | | -> number
 | | | | | |    "0"
 | | | | -> space
 | | | |    " "
 | | | | -> block
 | | | | | -> space
 | | | | |    "\n    "
 | | | | | -> declaration
 | | | | | | -> property
 | | | | | | | -> ident
 | | | | | | |    "opacity"
 | | | | | | -> propertyDelimiter
 | | | | | |    ":"
 | | | | | | -> space
 | | | | | |    " "
 | | | | | | -> value
 | | | | | | | -> number
 | | | | | | |    "0"
 | | | | | -> declarationDelimiter
 | | | | |    ";"
 | | | | | -> space
 | | | | |    "\n  "
 | | | -> space
 | | |    "\n  "
 | | | -> ruleset
 | | | | -> keyframesSelector
 | | | | | -> percentage
 | | | | | | -> number
 | | | | | |    "100"
 | | | | -> space
 | | | |    " "
 | | | | -> block
 | | | | | -> space
 | | | | |    "\n    "
 | | | | | -> declaration
 | | | | | | -> property
 | | | | | | | -> ident
 | | | | | | |    "opacity"
 | | | | | | -> propertyDelimiter
 | | | | | |    ":"
 | | | | | | -> space
 | | | | | |    " "
 | | | | | | -> value
 | | | | | | | -> number
 | | | | | | |    "1"
 | | | | | -> declarationDelimiter
 | | | | |    ";"
 | | | | | -> space
 | | | | |    "\n  "
 | | | -> space
 | | |    "\n"
```

See those `keyframesSelector` nodes? I've made them [according to spec](http://www.w3.org/TR/css3-animations/#keyframes).
The question is: do you need them to be wrapped in `selector` nodes?
In other words, could you choose one of the two examples?

```
# Example 1
| -> ruleset
| | -> keyframesSelector
| | | -> percentage
| | | | -> number
| | | |    "100"
```

```
# Example 2
| -> ruleset
| | -> selector
| | | -> keyframesSelector
| | | | -> percentage
| | | | | -> number
| | | | |    "100"
```

I feel like implementing the second one.

**Comment by bgriffith** [comment]
Happy to go with the second one.

**Comment by jwilsson** [comment]
Yeah, my vote is for the second one as well.

**Comment by tonyganch** [comment]
Fixed in @3.1.0
